proc-macros/src/lib.rs

```diff
@@ -189,6 +191,14 @@ fn build_client_impl(api: &api_def::ApiDefinition) -> Result<proc_macro2::TokenS
 fn build_client_functions(api: &api_def::ApiDefinition) -> Result<Vec<proc_macro2::TokenStream>, syn::Error> {
 	let visibility = &api.visibility;
 
+	let _crate = match (crate_name("jsonrpsee-http-client"), crate_name("jsonrpsee-ws-client")) {
```

Comment:
this is a hack just to find a client crate to get the types from, not sure if there's a nicer way to do without explicitly import the types crate.